Assets/HoloToolkit/Utilities/README.md

```diff
@@ -14,6 +14,16 @@ A drop in direction indicator that stays in the users view at all times.
 #### HeadsUpDirectionIndicatorPointer.prefab
 A quad based pointer to be used with the HeadsUpDirectionIndicator prefab to create an out of box direction indicator.
 
+###[Scenes](Scenes)
+---
+
+####HeadsetAdjustment.unity
```

Comment:
@NeerajW If you don't agree with my argument below, I'll move this under Utilities\Tests\Scenes as requested.

Having an entire scene as a building block is new to the HoloToolkit, but I feel that putting it under Utilities/Scenes is the right thing to do. 

I put it under Scenes because it is not a test scene for a prefab or a script, but is supposed to be used as a whole scene. The Utilities\Tests\Scenes would suggest that this is an example or test playground on how to use another part of the toolkit. But HeadsetAdjustment is a stand alone building block. 

Developers are not forced to use it, they can select what scenes are in their project in the Build Settings dialog. It is not even included by default.